acsoo/config.py

```diff
@@ -47,7 +47,7 @@ def series(self):
         r = self.__cfg.get(SECTION, "series")
         if not r:
             raise click.ClickException("Missing series in {}.".format(self.__cfgfile))
-        if r not in ("8.0", "9.0", "10.0", "11.0", "12.0", "13.0", "14.0", "15.0"):
+        if r not in tuple(str(float(r)) for r in range(8, 16 + 1)):
```

Comment:
I would have let the other version.

Prefer explicit things (in this case).

Moreover, you could put a variable instead for readability ?

Reply:
Yes it's why I put this change into a second commit, to have a easy-revert.